solidity/contracts/bridge/Bridge.sol

```diff
@@ -1312,8 +1310,87 @@ contract Bridge is Ownable, EcdsaWalletOwner {
             processedRedemptionsCount > 0,
             "Redemption transaction must process at least one redemption"
         );
+    }
+
+    /// @notice Processes a single redemption transaction output. Tries to
+    ///         identify output as a pending redemption request or reported
+    ///         redemption timeout. Output script passed to this function must
+    ///         not be the change output. Such output needs to be identified
+    ///         separately before calling this function.
+    ///         Reverts if outut is neither requested pending redemption nor
+    ///         requested and reported timed-out redemption.
+    ///         This function also marks each request as processed by removing
+    ///         them from `pendingRedemptions` mapping.
+    /// @param walletPubKeyHash 20-byte public key hash (computed using Bitcoin
+    //         HASH160 over the compressed ECDSA public key) of the wallet which
+    ///        performed the redemption transaction.
+    /// @param outputScript Non-change output script to be processed
+    /// @param outputValue Value of the output being processed
+    function processNonChangeRedemptionTxOutput(
```

Comment:
I think we should document return parameters and make it super-explicit that they are both `0` for timed out redemptions and they are not counted into the `totalBurnableValue` and `totalTreasuryFee`. At a first sight, I thought we changed the original behavior here.

Reply:
027b0bab7715569acf2430189f0b3bf1c14ec3d4